eva-remapping-ingest/src/test/java/uk/ac/ebi/eva/remapping/ingest/batch/io/RemappedSubmittedVariantsWriterTest.java

```diff
@@ -56,7 +55,6 @@
 @RunWith(SpringRunner.class)
 @ContextConfiguration(classes = {BatchTestConfiguration.class})
 @TestPropertySource("classpath:ingest-remapped-variants.properties")
-@UsingDataSet(locations = {"/test-data/submittedVariantEntity.json"})
```

Comment:
Also changed the tests so they don't rely on this dataset, as I find it easier to follow this way...